texar/torch/run/executor.py

```diff
@@ -332,6 +332,17 @@ def forward(self, batch):
         The device on which the model and data should be placed. Defaults to
         `None`, in which case GPUs will be used if available.
 
+    .. _executor-tbx-logging-args:
+
+    **Arguments for tensorboardX logging:**
+
+    `tbx_logging_dir`: str
+        Path to the directory for storing tensorboard logs.
+
+    `tbx_log_every`: |Condition|
+        Conditions that, when triggered, saves the tensorboard logs for train
+        metrics.
```

Comment:
Please mention that `log_every` will be used if `None` is provided. Also, the `.. _` marks an in-page anchor, please add an entry to the list of grouped parameters (here: https://github.com/asyml/texar-pytorch/blob/bbaf278e19ce153bb90546b47e9c9d0dbc475fca/texar/torch/run/executor.py#L256-L263).